### PR TITLE
19871: CT 116: Update Contact Details section of the VET TEC Profile Page

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a comparison tool user, I need the the contact details information to be displayed in a clear and concise manner so that I can easily identify the information I need when contacting a provider or SCO.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19871

## Testing done
Manual, Unit, and E2E tests pass locally

## Screenshots
Screen Shot 2019-11-01 at 3 13 09 PM

## Acceptance criteria
- [x] The physical address of the provider is displayed.
- [x] The mailing address of the provider is displayed.
- [x] The email address of the provider is displayed.
- [x] The phone number of the provider is displayed.
- [x] There is a heading for school certifying officials:
      The heading is displayed if the provider has SCOs.
      The heading is not displayed if the provider has no SCOs.
- [x] All primary SCOs are displayed together with the following information:
    Name (First Last)
    Title
    Email
    Phone Number
- [x] All secondary SCOs are displayed together with the following information:
    Name (First Last)
    Title
    Email
    Phone Number
- [x] If there is no data for AC 1-4, then the corresponding label is not displayed.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/vet-tec/VetTecContactInformation.jsx [hunk 1]
```diff
@@ -3,62 +3,175 @@ import PropTypes from 'prop-types';
...
+                {institution.physicalCity}, {institution.physicalState}{' '}
+                {institution.physicalZip}
+              </div>
             </div>
           </div>
         </div>
       )}
-  </div>
-);
+      {institution.physicalAddress1 && (
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecContactInformation.jsx]: This should be institution.address1, right?

File: src/applications/gi/components/vet-tec/VetTecContactInformation.jsx [hunk 2]
```diff
@@ -3,62 +3,175 @@ import PropTypes from 'prop-types';
...
-      institution.versionedSchoolCertifyingOfficials &&
-      institution.versionedSchoolCertifyingOfficials.length > 0 && (
-        <div>
-          <div className="vads-u-margin-top--4">
-            <h3>School certifying officials</h3>
+    );
+  }
+  return (
+    <div>
+      {institution.physicalAddress1 && (
```
Comment (zurbergram) [inline src/applications/gi/components/vet-tec/VetTecContactInformation.jsx]: can you break from this line 58 to line 137 out into a function (let's say `renderContactDetails`) that calls a function for each piece of information (i.e. `physicalAddress`, `mailingAddress`, `emailAddress`, `phoneNumber`)?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

**Review by zurbergram — CHANGES_REQUESTED** [review]
Please improve readability

File: src/applications/gi/components/vet-tec/VetTecContactInformation.jsx (hunk 1 above)
Comment (elder-joseph) [reply src/applications/gi/components/vet-tec/VetTecContactInformation.jsx]: That is correct, pushing an updated commit for this soon, once I make Matt's recommended changes.

**Review by zurbergram — APPROVED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**